src/Samples/Sample.AspNetCore/Controllers/CheckOutController.cs

```diff
@@ -114,6 +122,41 @@ public async Task<CardPayment> CreateCardPayment()
                 return null;
             }
         }
+        
+
+
+        public async Task<TrustlyPayment> CreateTrustlyPayment()
+        {
+            var totalAmount = this.cartService.CalculateTotal();
+            var vatAmount = Amount.FromDecimal(0);
+            try
+            {
+                var trustlyPaymentRequest = new TrustlyPaymentRequest(new CurrencyCode("SEK"),
+                                                                                     new List<Price>
+                                                                                     {
+                                                                                         new Price(Amount.FromDecimal(totalAmount),
+                                                                                                   PriceType.Trustly, vatAmount)
+                                                                                     },
+                                                                                     "Test Purchase", this.payeeInfoOptions.PayeeReference, "useragent", CultureInfo.GetCultureInfo("sv-SE"),
+                                                                                     new Urls(this.urls.HostUrls, this.urls.CompleteUrl,
+                                                                                              this.urls.TermsOfServiceUrl, this.urls.CancelUrl,
+                                                                                              this.urls.PaymentUrl, this.urls.CallbackUrl, this.urls.LogoUrl),
+                                                                                     new PayeeInfo(this.payeeInfoOptions.PayeeId,
+                                                                                                   this.payeeInfoOptions.PayeeReference), new SwedbankPay.Sdk.Payments.TrustlyPayments.PrefillInfo("Ola", "Nordmann"));
```

Comment:
Sending `Ola Nordmann` seems a bit weird here.
Could we instead use `TestDataService` and the information within there to prefil information so we use the same data for prefilling in the Sample project as we do in our tests?

Reply:
Then the Sample site would need a reference to the SystemTests projects. This doesn't make sense as this would result in circular references. Isn't it better to just remove `Ola Nordmann` then?

Reply:
Remove the test data, it's the simplest solution.

Reply:
Should probably be able to not pass any PrefilInfo, instead of creating an empty object. I will take a look at this and remove Ola Nordmann as prefilled value

Reply:
Extracting test data to a separate project is also an alternative.